test/e2e/login.spec.ts

```diff
@@ -0,0 +1,16 @@
+import { test, expect } from '@playwright/test';
+
+test('Login w User 1', async ({ page }) => {
+  await page.goto('/login?redirectTo=%2F');
+  await page.getByRole('button', { name: 'Sign In' }).click();
+  await page.getByLabel('Username').click();
+  await page.getByLabel('Username').fill('user1');
+  await page.getByLabel('Password').click();
+  await page.getByLabel('Password').fill('user1pass');
+  await page.getByRole('button', { name: 'Login' }).click();
+  await page.waitForURL(/.*\/$/);
```

Comment:
Can you use something better here?  It should be `http://localhost:8080/...` right?  Using a regex like this is a bit risky.